### ISSUE TITLE
unable to get output for regional studies

### ISSUE THREAD

**Issue by akashsinghraghuvanshi** [daemonkennett/ar_detection#1]
I was using your code to detect ARs for my region (lat: 38 -7, lon: 64-98) for the years 2017, 2018, and 2019 for 6,7,8,9 months. The code was successfully managed to run on the Jupyter notebook in Windows. But it is unable to give ar snapshots, characteristics, and ar masks. Can you please help me out in this matter as I am new to python and jupyter notebook.

**Comment by Jango96** [comment]
Yeah same here, has anyone got any Idea here? Any other solutions for the problem

**Comment by bahi696** [comment]
I also have the same problem, I got some ar_snapshots but somewhere out of the defined area just showing some wind directions:
![ (-ou-UTC_snapshot_3_1](https://github.com/daemonkennett/ar_detection/assets/80366272/50616a3a-5276-402c-8fc3-0b64b01bbc00)
, anyone finds the solution?
